src/H5Dpublic.h

```diff
@@ -677,7 +678,7 @@ H5_DLL herr_t H5Dget_chunk_info_by_coord(hid_t dset_id, const hsize_t *offset, u
  *                  indicating failure.
  *
  */
-H5_DLL herr_t H5Dchunk_iter(hid_t dset_id, H5D_chunk_iter_op_t cb, void *op_data);
+H5_DLL herr_t H5Dchunk_iter(hid_t dset_id, hid_t dxpl_id, H5D_chunk_iter_op_t cb, void *op_data);
 
```

Comment:
Is it not an API change? should not you do a v2?

Reply:
No, this was only merged to develop a week ago and there's no BW compatibility issues.